hackathon_site/dashboard/frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -69,6 +69,7 @@ const Dashboard = () => {
             <EditTeam
                 teamCode={team_code == null ? "None" : team_code}
                 canChangeTeam={true}
+                canLeaveTeam={team_size !== 1}
```

Comment:
I think this is a time where we leave the constraint handling to the backend because making all the warning/info messages on why someone can't leave the team is too much work & its not the main focus of the application. Focus on making the error handling really good and it will simplify this ticket.
In this case, you can remove the `canLeaveTeam` prop

Reply:
By having this condition checked, we can avoid some meaningless API requests. And it is intuitive because we know if we only have 1 ppl in the team, we are not allowed to leave the team.

Reply:
If you want to do that, you should also disable that button if the team has any orders.

Reply:
then I agree that handle this from the backend will be much easier! I will remove the "canLeaveTeam” check